Nitrox.Test/Nitrox.Test.csproj

```diff
@@ -10,10 +10,12 @@
         <PackageReference Include="AutoBogus" Version="2.13.1" />
         <PackageReference Include="AutoBogus.NSubstitute" Version="2.13.1" />
         <PackageReference Include="CompareNETObjects" Version="4.78.0" />
-        <PackageReference Include="coverlet.collector" Version="3.1.0" />
-        <PackageReference Include="FluentAssertions" Version="6.2.0" />
-        <PackageReference Include="MSTest" Version="3.0.2" />
-        <PackageReference Include="NSubstitute" Version="4.2.2" />
+        <PackageReference Include="FluentAssertions" Version="6.9.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+        <PackageReference Include="MSTest.TestAdapter" Version="3.0.2" />
+        <PackageReference Include="MSTest.TestFramework" Version="3.0.2" />
```

Comment:
Why are you splitting up the MSTest reference here? Also this isn't ordered alphabetically anymore.

Reply:
Do we *manually* have to keep it alphabetic? All the changes I did were done through nuget package manager.